backend/danswer/danswerbot/slack/blocks.py

```diff
@@ -360,22 +359,6 @@ def build_quotes_block(
     return [SectionBlock(text="*Relevant Snippets*\n" + "\n".join(quote_lines))]
 
 
-def build_standard_answer_blocks(
```

Comment:
I'm honestly not sure why this is removed; maybe stale code?

Reply:
was moved to the ee equivalent module I believe